### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/demo.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#792] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/demo.html

让你的 VuePress 站点中的 Markdown 文件支持代码案例。 配置 语法 请使用以下语法: 提示 JSON 块是可选的，可用的配置详见 。 该插件支持三种类型: normal vue react 可用的语言 你可以在演示块中使用不同语言。 当你设置一些不能在浏览器上直接运行的语言时，由于插件无法解析它们，因此网页演示将被禁用。插件只显示代码...

https://theme-hope.vuejs.press/zh/guide/markdown/demo.html

**Comment by dcbestwords** [comment]
我演示vue代码的时候，style标签如果使用预处理语言，比如less就会失效，我该怎么引入less
